iaf-commons/src/main/java/nl/nn/adapterframework/util/Environment.java

```diff
@@ -0,0 +1,80 @@
+/*
+   Copyright 2023 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Environment {
```

Comment:
Dit zou ook PropertiesUtil kunnen zijn.
Een methode om properties te lezen kan namelijk handig zijn. We hebben nu op een aantal plekken:
```
try(FileInputStream fis = new FileInputStream(propertiesFile); Reader reader = StreamUtil.getCharsetDetectingInputStreamReader(fis)) {
	properties.load(reader);
}
```
Misschien dat hier dan ook StringSubstitutions in kunnen?